### ISSUE TITLE
Extract Genomic DNA error

### ISSUE THREAD

**Issue by igormakunin** [gvlproject/gvl_image_playbook#51]
Fetch Sequences > "Extract Genomic DNA using coordinates from assembled/unassembled genomes" does not work with available genomes (tested on hg19). 

The tool: /mnt/galaxy/galaxy-app/tools/extract/extract_genomic_dna.py

**Comment by igormakunin** [comment]
Galaxy-qld users have errors with the tool, so I've tried to fix it. It needs 2bit fasta. GVL-4.1.0 does not provide 2bit. I've installed data_manager_twobit_builder and created 2bit fasta on a test GVL-4.1.0 instance. Galaxy was restarted, but the tool failed: No sequences are available for 'hg19', request them by reporting this error. The fix: .loc files alignseq.loc, lastz_seqs.loc and twobit.loc were copied from /mnt/galaxyIndices/tool-data/dm/toolshed.g2.bx.psu.edu/repos/devteam/data_manager_twobit_builder/74b09c8e5f6e/ to /mnt/galaxyIndices/. Derek says the tool is 'native' to Galaxy, so it does not see path to loc file created for toolshed tools. 
Is it a proper way to fix the error?
Can we add 2bit to indices? This will enable Extract Genomic DNA and lastz tools, with proper .loc files.
